### PR TITLE
Preparation for IContainerRuntimeOptions.enableGroupedBatching removal

### PR DESCRIPTION
**Pull request by MarioJGMsoft**
## Description

This PR is a preparation for the breaking change that will happen when we remove IContainerRuntimeOptions.enableGroupedBatching 

## Breaking Changes

Link to the breaking issue: #[23270](https://github.com/microsoft/FluidFramework/issues/23270)

## Reviewer Guidance

- Let me know if I missed something or if there's a better solution for what I want to accomplish

Owning Task: [AB#26793](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/26793)

### CODE REVIEW

File: packages/test/test-end-to-end-tests/src/test/benchmark/opCriticalPath.time.spec.ts [hunk 1]
```diff
@@ -19,19 +20,21 @@ import {
...
-const testContainerConfig: ITestContainerConfig = {
-	runtimeOptions: {
-		enableGroupedBatching: true,
-		summaryOptions: {
-			initialSummarizerDelayMs: 0, // back-compat - Old runtime takes 5 seconds to start summarizer without thi
-			summaryConfigOverrides: {
-				...DefaultSummaryConfiguration,
-				...{ maxOps: 10, initialSummarizerDelayMs: 0, minIdleTime: 10, maxIdleTime: 10 },
-			},
+const runtimeOptions: IContainerRuntimeOptionsInternal = {
```
Comment (markfields) [inline packages/test/test-end-to-end-tests/src/test/benchmark/opCriticalPath.time.spec.ts:23]: Shouldn't be necessary, `ITestContainerConfig.runtimeOptions` is already the internal type

Comment (MarioJGMsoft) [reply packages/test/test-end-to-end-tests/src/test/benchmark/opCriticalPath.time.spec.ts]: Added changes

**Review by noencke — APPROVED** [review]

File: packages/runtime/container-runtime/src/test/containerRuntime.spec.ts [hunk 2]
```diff
@@ -2593,9 +2594,7 @@ describe("Runtime", () => {
 					registryEntries: [],
 					existing: false,
 					requestHandler: undefined,
-					runtimeOptions: {
-						enableGroupedBatching: false,
-					},
+					runtimeOptions: {},
```
Comment (markfields) [inline packages/runtime/container-runtime/src/test/containerRuntime.spec.ts:2597]: This and the next one look like behavior changes to me - switching from `false` to the default value `true`

Comment (MarioJGMsoft) [reply packages/runtime/container-runtime/src/test/containerRuntime.spec.ts]: I missed that, thanks for pointing it out!

**Review by markfields — APPROVED** [review]
LGTM!

**Pull request merged**